streams/dlna_meta.py

```diff
@@ -2,22 +2,37 @@
 import sys
 import time
 import re
+import argparse
+import os
 
 # parse the log file generated by gmrender-resurrect, extracting out the track info
 
 # TODO: dlna generates state update events to subscribers, in the future we should use this method instead
 # Check out https://github.com/hzeller/upnp-display/blob/master/controller-state.cc for details
 
-args = sys.argv[1:]
-print('Input Arguments: {}'.format(args[0]))
-try:
-    src = int(args[0])
-    loc = '/home/pi/config/dlna/{}/'.format(src)
-except:
-    print('Error: Invalid source choice. Sources range from 0 to 3. Please try again.')
-    sys.exit('Failure.')
-print('Targeting {}'.format(loc))
-cs_loc = loc + 'currentSong'
+# args = sys.argv[1:]
```

Comment:
probably should remove this commented out code since it looks like there is a better version below. If it makes sense to keep it, add a comment explaining why